src/test/runTest.ts

```diff
@@ -18,7 +19,8 @@ async function main() {
     // Passed to --extensionTestsPath
     const extensionTestsPath = path.resolve(__dirname, "./suite/index");
 
-    const vscodeExecutablePath = await downloadAndUnzipVSCode();
+    const vscodeVersion = env.VSCODE_VERSION === "legacy" ? "1.66.0" : "stable";
```

Comment:
I figured two versions back was good enough.  We can bump this one in the future if we want